aws/rust-runtime/aws-config/test-data/profile-provider/credential_process/fs/home/.aws/config

```diff
@@ -1,6 +1,7 @@
 [default]
 source_profile = base
-credential_process = echo '{ "Version": 1, "AccessKeyId": "ASIARTESTID", "SecretAccessKey": "TESTSECRETKEY", "SessionToken": "TESTSESSIONTOKEN", "Expiration": "2022-05-02T18:36:00+00:00" }'
+# base64 encoded to prevent triggering the secret scanner
```

Comment:
Reworded to make this sound less scary:
```suggestion
# these fake credentials are base64 encoded to prevent triggering the unit test that scans logs for secrets
```